From 11d9ed2e849b93843d22b2e088c1a45f9fd02b27 Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Thu, 11 Jul 2024 23:17:23 -0700
Subject: [PATCH] intel-ice: use linux common header for kcompat-generator.sh

For debian, there are two header pkgs: linux-headers-5.10.0-6-amd64
and linux-headers-5.10.0-6-common. kcompat-generator.sh needs
header files in linux-headers-5.10.0-6-common instead of
linux-headers-5.10.0-6-amd64.

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 src/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common.mk b/src/common.mk
index 52d2247..0575616 100644
--- a/src/common.mk
+++ b/src/common.mk
@@ -346,7 +346,7 @@ minimum_kver_check = $(eval $(call _minimum_kver_check,${1},${2},${3}))
 # generated also via $(shell) call, which makes error handling ugly
 $(if $(shell \
     $(if $(findstring 1,${V}),,QUIET_COMPAT=1) \
-    KSRC=${KSRC} OUT=${src}/kcompat_generated_defs.h CONFFILE=${CONFIG_FILE} \
+    KSRC=${KSRC_COMMON} OUT=${src}/kcompat_generated_defs.h CONFFILE=${CONFIG_FILE} \
     bash ${src}/kcompat-generator.sh && echo ok), , $(error kcompat-generator.sh failed))
 
 KCOMPAT_DEFINITIONS := $(shell ${CC} ${EXTRA_CFLAGS} -E -dM \
-- 
2.43.0

